Code4SocialGood/C4SG
# [refactoring] angular cli migration

[Angular Cli](https://github.com/angular/angular-cli) alleviates the pain of manual webpack configuration, in addition to providing support for AOT compilation, lazy loading modules, `ng generate` support for creating Angular objects, and well you'll just have to check out the code and the docs to learn everything!

To test

- `rm -rf node_modules/`
- `npm install`
- `npm start`

And navigate to `localhost:4200` and confirm the site loads! I am currently trying to hack the images to work again, and testing to make sure no functionality has been lost.

Once this is merged, I will work on tests: configuring and fixing.

Please ask questions or offer comments.
